src/commcare_cloud/ansible/roles/elasticsearch/defaults/main.yml

```diff
@@ -19,3 +19,5 @@ elasticsearch_service_name: elasticsearch
 elasticsearch_node_name: "{{ hostname }}"
 elasticsearch_delete_old_logs: False
 elasticsearch_log_retention_days: 7
+# to test plugin on staging only
+is_staging: False
```

Comment:
I think this should be renamed to `es_plugins_phonetic_enabled` or something like that.